### ISSUE TITLE
[Company Cards] Card number is not visible in the cards table 

### ISSUE THREAD

**Issue by mountiny** [Expensify/App#52915]
Coming from [here](https://expensify.slack.com/archives/C06ML6X0W9L/p1732190694256039?thread_ts=1732108983.281759&cid=C06ML6X0W9L)

When connecting a direct feed connection, we are missing the last 4 pan number in the company cards feed table.

![image](https://github.com/user-attachments/assets/4bd44352-4771-4792-a28c-7730ede26c19)

**Comment by VickyStash** [comment]
The issue will be fixed in this PR: https://github.com/Expensify/App/pull/52893#issuecomment-2491312432

**Comment by isabelastisser** [comment]
@VickyStash, does this mean we can close this issue because it will be fixed when the other issue is fixed?

**Comment by mountiny** [comment]
Yes we can let this open until the pr is deployed to production and then we can close it as we will handle the regression tests addition with Joe for the project altogether

**Comment by mountiny** [comment]
# BugZero Checklist:

- [ ] **[Contributor]** Classify the bug:

<details>
<summary>Bug classification</summary>


Source of bug:
  - [ ] 1a. Result of the original design (eg. a case wasn't considered)
  - [ ] 1b. Mistake during implementation
  - [ ] 1c. Backend bug
  - [ ] 1z. Other:

Where bug was reported:
  - [ ] 2a. Reported on production
  - [ ] 2b. Reported on staging (deploy blocker)
  - [ ] 2c. Reported on both staging and production
  - [ ] 2d. Reported on a PR
  - [ ] 2z. Other:

Who reported the bug:
  - [ ] 3a. Expensify user
  - [ ] 3b. Expensify employee
  - [ ] 3c. Contributor
  - [ ] 3d. QA
  - [ ] 3z. Other:

</details>

- [ ] **[Contributor]** The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake.

    Link to comment:

- [ ] **[Contributor]** If the regression was CRITICAL (e.g. interrupts a core flow) A discussion in [#expensify-open-source](https://app.slack.com/client/E047TPA624F/C01GTK53T8Q) has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner.

    Link to discussion:

- [ ] **[Contributor]** If it was decided to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps using the template below to ensure the same bug will not reach production again.

<details>
<summary>Regression Test Proposal Template</summary>


- [ ] **[BugZero Assignee]** Create a GH issue for creating/updating the regression test once above steps have been agreed upon.

    Link to issue:

## Regression Test Proposal
### Precondition:


-

### Test:


1.

Do we agree 👍 or 👎


</details>

**Comment by mountiny** [comment]
Noone to pay here now and no need for regression testing
